backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -1304,7 +1305,7 @@ def test_qnn_backend_profile_op(self):
             module,
             sample_input,
             expected_partitions=1,
-            expected_profile_events=25,
+            expected_profile_events=24,
```

Comment:
what is expected profile event?

Reply:
The test case "test_qnn_backend_profile_op" is checking the functionality of the etdump flow on simple test model. And the way we validate the etdump is that checking the index of event in the etdump file.

Previously, the etdump mechanism has some issue (That's why we "@unittest.expectedFailure" in Line 1292) , and it seems the issue have been fix now. After applying the fixed etdump flow, we found that the number of events for the simple test model is changed:

- The floating point case totally have 24 events
- The quantized one totally have 25 events

So we change the number here and also resolve the TODO item in "executor_runner/qnn_executor_runner.cpp" to enable the correct test flow.